OrbitCaptureClient/CaptureClient.cpp

```diff
@@ -50,7 +50,7 @@ ErrorMessageOr<void> CaptureClient::StartCapture(
 
   state_ = State::kStarting;
 
-  std::unique_ptr<ProcessData> process_copy = process.CreateCopy();
```

Comment:
You could just have the lambda capture `process` by value, instead of creating a copy and the moving the copy. Same goes with `module_map` and `selected_functions`.

Reply:
As, discussed offline, this does not compile as the process is passed as ref and capture will be the ref.
I will keep it as is.

Reply:
So yeah, just for your information, I had a quick look and&mdash;more or less what you said&mdash; the problem is that `process` is passed by const reference (which is how it's captured, I didn't know that), *and* that `Capture` requires a (non-const, obviously) r-value as first parameter. You can't make a const-ref an r-value. What one could do to be more concise it to make the copy right when passing the parameter, like
```
  thread_pool->Schedule(
      [this, process, module_map = std::move(module_map_copy),
       selected_functions = std::move(selected_functions), selected_tracepoints]() mutable {
        Capture(ProcessData{process}, std::move(module_map), std::move(selected_functions),
                std::move(selected_tracepoints));
      });
```
which saves one move, the one into the lambda.

Reply:
Would be fine for me. If you like, you can just create a small PR for that.
Thanks for looking into this.